java/serving-client/src/main/java/dev/feast/FeastClient.java

```diff
@@ -57,9 +58,9 @@ public class FeastClient implements AutoCloseable {
    * @param port port number of Feast serving GRPC server
    * @return {@link FeastClient}
    */
-  public static FeastClient create(String host, int port) {
+  public static FeastClient create(String host, int port, Deadline deadline) {
```

Comment:
Probably a good idea to make this backwards-compatible and provide some default value here as well.

Reply:
✅